packages/core/useEventSource/index.ts

```diff
@@ -115,8 +115,8 @@ export function useEventSource<Events extends string[]>(
   events: Events = [] as unknown as Events,
   options: UseEventSourceOptions = {},
 ): UseEventSourceReturn<Events> {
-  const event: Ref<string | null> = deepRef(null)
-  const data: Ref<string | null> = deepRef(null)
+  const event: Ref<string | null> = shallowRef(null)
+  const data: Ref<unknown> = deepRef(null)
   const status = shallowRef('CONNECTING') as Ref<EventSourceStatus>
   const eventSource = deepRef(null) as Ref<EventSource | null>
   const error = shallowRef(null) as Ref<Event | null>
```

Comment:
```suggestion
  const status = shallowRef<EventSourceStatus>('CONNECTING')
  const eventSource = deepRef<EventSource | null>(null)
  const error = shallowRef<Event | null>(null)
```

Reply:
same as the other comments, it was just already this way

ill update though